diff --git a/config.def.h b/config.def.h
index eebd324a01..c85df44e5c 100644
--- a/config.def.h
+++ b/config.def.h
@@ -460,7 +460,7 @@ static bool menu_show_quit_retroarch     = true;
 static bool menu_show_restart_retroarch  = true;
 static bool menu_show_reboot             = true;
 static bool menu_show_shutdown           = true;
-#if defined(HAVE_LAKKA) || defined(VITA) || defined(_3DS)
+#if defined(VITA) || defined(_3DS)
 static bool menu_show_core_updater       = false;
 #else
 static bool menu_show_core_updater       = true;
diff --git a/menu/menu_displaylist.c b/menu/menu_displaylist.c
index 4f9cb0f0bb..b3c34bd5e2 100644
--- a/menu/menu_displaylist.c
+++ b/menu/menu_displaylist.c
@@ -7823,37 +7823,7 @@ bool menu_displaylist_ctl(enum menu_displaylist_ctl_state type,
             settings_t *settings         = config_get_ptr();
 #endif
 
-#ifdef HAVE_LAKKA
-            if (menu_entries_append_enum(info->list,
-                     msg_hash_to_str(MENU_ENUM_LABEL_VALUE_UPDATE_LAKKA),
-                     msg_hash_to_str(MENU_ENUM_LABEL_UPDATE_LAKKA),
-                     MENU_ENUM_LABEL_UPDATE_LAKKA,
-                     MENU_SETTING_ACTION, 0, 0))
-               count++;
-
-            if (settings->bools.menu_show_legacy_thumbnail_updater)
-            {
-               if (menu_entries_append_enum(info->list,
-                        msg_hash_to_str(MENU_ENUM_LABEL_VALUE_THUMBNAILS_UPDATER_LIST),
-                        msg_hash_to_str(MENU_ENUM_LABEL_THUMBNAILS_UPDATER_LIST),
-                        MENU_ENUM_LABEL_THUMBNAILS_UPDATER_LIST,
-                        MENU_SETTING_ACTION, 0, 0))
-                  count++;
-            }
-
-            if (menu_entries_append_enum(info->list,
-                     msg_hash_to_str(MENU_ENUM_LABEL_VALUE_PL_THUMBNAILS_UPDATER_LIST),
-                     msg_hash_to_str(MENU_ENUM_LABEL_PL_THUMBNAILS_UPDATER_LIST),
-                     MENU_ENUM_LABEL_PL_THUMBNAILS_UPDATER_LIST,
-                     MENU_SETTING_ACTION, 0, 0))
-               count++;
-            if (menu_entries_append_enum(info->list,
-                     msg_hash_to_str(MENU_ENUM_LABEL_VALUE_DOWNLOAD_CORE_CONTENT),
-                     msg_hash_to_str(MENU_ENUM_LABEL_DOWNLOAD_CORE_CONTENT_DIRS),
-                     MENU_ENUM_LABEL_DOWNLOAD_CORE_CONTENT_DIRS,
-                     MENU_SETTING_ACTION, 0, 0))
-               count++;
-#elif defined(HAVE_NETWORKING)
+#if defined(HAVE_NETWORKING)
 #if defined(__WINRT__) || defined(WINAPI_FAMILY) && WINAPI_FAMILY == WINAPI_FAMILY_PHONE_APP
 #else
             if (settings->bools.menu_show_core_updater)
