diff --git a/config.def.h b/config.def.h
index eebd324a01..c85df44e5c 100644
--- a/config.def.h
+++ b/config.def.h
@@ -460,7 +460,7 @@ static bool menu_show_quit_retroarch     = true;
 static bool menu_show_restart_retroarch  = true;
 static bool menu_show_reboot             = true;
 static bool menu_show_shutdown           = true;
-#if defined(HAVE_LAKKA) || defined(VITA) || defined(_3DS)
+#if defined(VITA) || defined(_3DS)
 static bool menu_show_core_updater       = false;
 #else
 static bool menu_show_core_updater       = true;
diff --git a/menu/drivers/materialui.c b/menu/drivers/materialui.c
index b01bb0f89b..c473806af3 100644
--- a/menu/drivers/materialui.c
+++ b/menu/drivers/materialui.c
@@ -5142,7 +5142,7 @@ static int materialui_list_push(void *data, void *userdata,
 #ifdef HAVE_LAKKA
             entry.enum_idx      = MENU_ENUM_LABEL_UPDATE_LAKKA;
             menu_displaylist_setting(&entry);
-#else
+#endif
             {
                if (settings->bools.menu_show_online_updater)
                {
@@ -5150,7 +5150,6 @@ static int materialui_list_push(void *data, void *userdata,
                   menu_displaylist_setting(&entry);
                }
             }
-#endif
 
             if (settings->bools.menu_content_show_netplay)
             {
-- 
2.17.1

